celestia/peers.txt

```diff
@@ -1 +0,0 @@
-d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
```

Comment:
wait is the peer ID already in the peers.txt?

Reply:
we want to avoid conflicts in this file, and can do that by not changing it relative to main

Reply:
Not our peer and dont know why it was in there, we submitted only one see https://github.com/celestiaorg/networks/pull/382/commits/3038515473763345c2b5b781f0a2bdbefd9d7484

Reply:
If not ok i can close this PR and make new

Reply:
the important thing is to not modify the peers.txt file relative to the master branch. If that occurs, there will likely be a conflict. one way to make sure there isn't a change is to copy the file from the master branch and paste it in this one. That or just revert any changes to that file.